### PR TITLE
Correct Event Action to include endgame event schema

### PR DESCRIPTION
**Pull request by shashank-elastic**
## Issues
#2603 

## Summary
Removing endgame logs as they had testing gaps and was originally not scoped for the rule development

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by imays11 — APPROVED** [review]
👍🏾

**Review by brokensound77 — APPROVED** [review]
LGTM - removing due to inconsistency of `exec` event across datasources

**Comment by brokensound77** [comment]
> LGTM - removing due to inconsistency of exec event across datasources

Having a better understanding of the issue, it is more appropriate to just add the endgame specific value

File: rules/linux/credential_access_credential_dumping.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ for password-cracking utilities or prepare themselves for future operations by g
 credential information of the victim.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "endgame-*"]
+index = ["logs-endpoint.events.*"]
```
Comment (brokensound77) [inline rules/linux/credential_access_credential_dumping.toml:20]:
```suggestion
index = ["logs-endpoint.events.*", "endgame-*"]
```

Comment (brokensound77) [reply rules/linux/credential_access_credential_dumping.toml:20]: also, update the query to

```diff
process where process.name == "unshadow" and
- event.type == "start" and event.action == "exec" and process.args_count >= 2
+ event.type == "start" and event.action in ("exec", "exec_event") and process.args_count >= 2
```

File: rules/linux/credential_access_credential_dumping.toml
Comment (w0rk3r) [inline rules/linux/credential_access_credential_dumping.toml:1]: After bumping the updated_date, LGTM

**Review by w0rk3r — APPROVED** [review]

**Pull request merged**